config/php.conf.ini

```diff
@@ -2,4 +2,6 @@ file_uploads = On
 memory_limit = 64M
 upload_max_filesize = 64M
 post_max_size = 64M
-max_execution_time = 600
\ No newline at end of file
+max_execution_time = 600
+log_errors = On
+error_log = /dev/stderr
```

Comment:
Should this be here?

Reply:
This allows error logging to actually happen, I don't believe it changes anything for the user

I was a bit surprised that it wasnt there already actually, how did everybody else debug?

Reply:
I set up XDebug in my vs code.

Reply:
I am still not using the docker setup. I look at the debug log for Wordpress generally which doesn't require this (I don't think).